app/assets/javascripts/discourse/components/image-uploader.js.es6

```diff
@@ -1,9 +1,39 @@
-import computed from "ember-addons/ember-computed-decorators";
+import {
+  default as computed,
+  observes
+} from "ember-addons/ember-computed-decorators";
+
 import UploadMixin from "discourse/mixins/upload";
+import lightbox from "discourse/lib/lightbox";
+import { ajax } from "discourse/lib/ajax";
+import { popupAjaxError } from "discourse/lib/ajax-error";
 
 export default Ember.Component.extend(UploadMixin, {
   classNames: ["image-uploader"],
-  infoHidden: true,
+
+  init() {
+    this._super(...arguments);
+    this._applyLightbox();
+  },
+
+  willDestroyElement() {
+    this._super(...arguments);
+    $("a.lightbox").magnificPopup("close");
+  },
+
+  @observes("imageUrl")
+  _applyLightbox() {
+    if (this.get("imageUrl")) {
+      Ember.run.next(() => {
+        lightbox(this.$());
+      });
+    }
+  },
+
+  @computed("imageWidth", "imageHeight", "imageFilesize")
+  imageInfo(imageWidth, imageHeight, imageFilesize) {
```

Comment:
It looks like this could be done in the template itself without a computed property.